### PR TITLE
zh: sync concepts/cluster-administration/addons

### PR DESCRIPTION
**Pull request by chenrui333**
relates to #27797
also relates to previous PR, #27797

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Add-ons 扩展了 Kubernetes 的功能。
...
 * [Contiv](http://contiv.github.io) provides configurable networking (native L3 using BGP, overlay using vxlan, classic L2, and Cisco-SDN/ACI) for various use cases and a rich policy framework. Contiv project is fully [open sourced](http://github.com/contiv). The [installer](http://github.com/contiv/install) provides both kubeadm and non-kubeadm based installation options.
 * [Contrail](http://www.juniper.net/us/en/products-services/sdn/contrail/contrail-networking/), based on [Tungsten Fabric](https://tungsten.io), is an open source, multi-cloud network virtualization and policy management platform. Contrail and Tungsten Fabric are integrated with orchestration systems such as Kubernetes, OpenShift, OpenStack and Mesos, and provide isolation modes for virtual machines, containers/pods and bare metal workloads.
-* [Flannel](https://github.com/coreos/flannel/blob/master/Documentation/kubernetes.md) is an overlay network provider that can be used with Kubernetes.
+* [Flannel](https://github.com/flannel-io/flannel#deploying-flannel-manually) is an overlay network provider that can be used with Kubernetes.
```
Comment (howieyuen) [inline content/zh/docs/concepts/cluster-administration/addons.md:36]: please update zh link as well

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/concepts/cluster-administration/addons.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/concepts/cluster-administration/addons.md:36]: Yes, please update line 66 in this file.

Comment (chenrui333) [reply content/zh/docs/concepts/cluster-administration/addons.md:36]: sorry about the delay, just finished the update of l66

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**